systemd/authd.service

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=Authd daemon service
+After=authd.socket
+Requires=authd.socket
+PartOf=authd.socket
+
+[Service]
+Type=notify
+ExecStart=/sbin/authd
+
+# Some daemon restrictions
+CapabilityBoundingSet=
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=true
+PrivateDevices=yes
+PrivateMounts=yes
+PrivateNetwork=yes
```

Comment:
I am concerned about that one if sss is configured on the system. Remember that for cleaning up users from system groups when the cache is corrupted, we are going to calling getent passwd to list users. Maybe sss is configured to call the ldap server here. Isn’t that going to be an issue?

Reply:
The getent user check is not yet implemented (part of pr #150)